crates/forge/src/main.rs

```diff
@@ -223,6 +223,7 @@ fn main_execution() -> Result<bool> {
             Ok(all_failed_tests)
         })
     })??;
+    rt.shutdown_background();
```

Comment:
Why do we need that? Don't we wait for all tasks to shutdown anyways?

Reply:
good catch, It has to be added during merge PRs :o